about.html

```diff
@@ -0,0 +1,161 @@
+          <div class="closeBtn"><i class="fas fa-times fa-2x"></i></div>
+        </div>
+        <nav>
+          <ul class="nav-ul">
+            <li><a class="menu-link" href="./index.html">Home</a></li>
+            <li><a class="menu-link" href="#program">Program</a></li>
+            <li><a class="menu-link" href="#">Join</a></li>
+            <li><a class="menu-link" href="#partner">Sponsors</a></li>
+            <li><a class="menu-link" href="#">News</a></li>
+          </ul>
+        </nav>
+        <button class="campaign">Campaign</button>
+      </div>
+    </header>
+    <main>
+      <section class="info">
+        <div class="menu">
+          <i class="fas fa-bars fa-2x"></i>
+        </div>
+        <h2>"Hello! Sharing world"</h2>
+        <h1>WORLD ANNUAL SPORTS CONFERENCE</h1>
+        <hr />
+        <div class="info-descr">
+          <p>
+            This page describes the World annual sports conference 
+            which holds annually. The aim of the conference is to
+            celebrate the impact of the sports in bringing different cultures 
+            and people together and also to discuss ways to move forward.
+          </p>
+        </div>
+        <div class="info-contact">
+          <h3>
+            Please contact us per Email for any further questions about
+             world annual sports conference.
+          </h3>
+          <h4>sports.concaf@rmd.org</h4>
+        </div>
+      </section>
+      <section class="the-logo">
+        <h2 class="section-title">
+          Creative Commons<br />World Sports 2022
+        </h2>
+        <hr />
+        <p>
+          The logo of World Sports Conference 2022 was decided through the
+          logo competition from 10 Jan to 13 Feb. After Online public voting and 
+          selection commitee's judging process, the logo from font-awesome won the prize.
+        </p>
+        <div class="logo-conf">
+          <div class="logopro">
+            <i class="fa-solid fa-baseball"></i>
+            <p>SPORTS</p>
+          </div>
+        </div>
+      </section>
+      <section class="past about">
+        <h2 class="section-title">See the past World Sports Conferences</h2>
+        <hr />
+        <p>
+          Take a look at the last two Sports Conferences which took place in
+          England and Germany
+        </p>
+        <div class="years">
+          <div class="year-box year-fst">
+            <h3 class="year">2020</h3>
+            <p>Sports Conference in England</p>
+          </div>
+          <div class="year-box year-sed">
+            <h3 class="year">2021</h3>
+            <p>Sports Conference in Germany</p>
+          </div>
+          <!-- <div class="year-box bott year-thd">
+            <h3 class="year">2020</h3>
+            <p>Facebook IG summit in Paris</p>
+          </div> -->
+        </div>
+      </section>
+    </main>
+    <footer>
+      <section class="partner" id="partner">
+        <h2>Partner</h2>
+        <hr />
```

Comment:
- [ ] Footer of about page should not have the logo container and should have a dark background:

Your design  | Behance design
------------- | -------------
<img width="670" alt="Screenshot 2022-03-10 at 13 26 57" src="https://user-images.githubusercontent.com/77038610/157631867-26690529-1a27-404d-8172-0d887a12b983.png"> | <img width="862" alt="Screenshot 2022-03-10 at 13 27 17" src="https://user-images.githubusercontent.com/77038610/157631947-d3a8a12d-b954-4b66-83a1-87487707081a.png">